pilot/prompts/prompts.py

```diff
@@ -37,6 +37,7 @@ def ask_for_app_type():
 
 def ask_for_main_app_definition(project):
     question = 'Describe your app in as much detail as possible.'
+    print('example project', type='button')
```

Comment:
do we want to then fill in project description?

Reply:
This crept in from a different experiment, not in scope for this PR.